astropy/coordinates/tests/test_transformations.py

```diff
@@ -45,9 +45,9 @@ def tfun(c, f):
     assert_allclose(c2.dec.radian, 0.5)
 
     def matfunc(coo, fr):
-        return [[1, 0, 0],
-                [0, coo.ra.degree, 0],
-                [0, 0, 1]]
+        return np.array([[1, 0, 0],
```

Comment:
My guess is that this change is no longer necessary, and I think the matrix now always ends up in a ufunc and thus is an `ndarray` is generated automatically.

Reply:
✅ 🔙